DataDefinitions/Body.cs

```diff
@@ -206,6 +206,8 @@ public Body()
 
         // Additional calculated star information
         [JsonIgnore]
+        public bool scoopable => "KGBFOAM".Contains(stellarclass ?? String.Empty);
```

Comment:
This evaluates to `true` if `stellarclass` is null or empty. Is that the intention?